web/src/components/tasks/anytime-action/tax-clearance-certificate/AnytimeActionTaxClearanceCertificateElement.test.tsx

```diff
@@ -673,6 +812,91 @@ describe("<AnyTimeActionTaxClearanceCertificateReviewElement />", () => {
     });
 
     describe("renders data when input is provided", () => {});
+
+    describe("renders errors on tax clearance step 2", () => {
```

Comment:
we should have an onBlur test for the address components it feels odd they are not firing since they are required

Reply:
I'm going to save this for the next ticket specifically related to the alert box